src/client/js/otp/config.js

```diff
@@ -71,27 +71,27 @@ otp.config = {
      */
 
     baseLayers: [
-        {
-            name: 'Stamen Terrain',
-            tileUrl: 'http://tile.stamen.com/terrain/{z}/{x}/{y}.png',
-            attribution : 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="http://www.openstreetmap.org/copyright">ODbL</a>.'
-        },
+        // {
+        //     name: 'Stamen Terrain',
+        //     tileUrl: 'http://tile.stamen.com/terrain/{z}/{x}/{y}.png',
+        //     attribution : 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="http://www.openstreetmap.org/copyright">ODbL</a>.'
+        // },
```

Comment:
I think we should delete this code, not just comment it out. We can always go back in Git to find it if we need to.

Reply:
I agree